docs/source/onedal/spmd/index.rst

```diff
@@ -0,0 +1,68 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
+.. *
+.. * Licensed under the Apache License, Version 2.0 (the "License");
+.. * you may not use this file except in compliance with the License.
+.. * You may obtain a copy of the License at
+.. *
+.. *     http://www.apache.org/licenses/LICENSE-2.0
+.. *
+.. * Unless required by applicable law or agreed to in writing, software
+.. * distributed under the License is distributed on an "AS IS" BASIS,
+.. * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. * See the License for the specific language governing permissions and
+.. * limitations under the License.
+.. *******************************************************************************/
+
+.. _dg_spmd:
+
+============================
+Single Program Multiple Data
+============================
+
+This section includes concepts and description of objects that support 
+distributed computations using :capterm:`SPMD` model.
+
+#. Distributed computation using SPMD model
```

Comment:
this markup is for introducing numbered lists, not clear to me why do you need a numbered list with one item here?